src/features/landing/components/AuthSection/SocialItem/SocialItem.tsx

```diff
@@ -7,13 +7,12 @@ import { UiTypography } from '@/components/ui';
 import { socialItemStyles } from './styles';
 
 interface SocialItemProps {
```

Comment:
need to move types in another file, in the types folder, or in a folder of the component